apps/web/pages/api/v1/environments/[environmentId]/surveys/[surveyId]/og/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { ImageResponse } from "next/server";
+import { NextRequest } from "next/server";
+
+export const dynamic = "auto";
+export const dynamicParams = true;
+export const revalidate = false;
+export const fetchCache = false;
+
+// App router includes @vercel/og.
+// No need to install it.
+
+export const runtime = "edge";
+
+export default async function GET(req: NextRequest) {
+  let name = req.nextUrl.searchParams.get("name");
+  let brandColor = req.nextUrl.searchParams.get("brandColor");
+
+  return new ImageResponse(
```

Comment:
This doesnt look like the mockup, will give it a try :)

Reply:
Oh, can you please specify the changes?